testutil/src/test/java/org/spine3/test/NullToleranceTestShould.java

```diff
@@ -140,14 +141,23 @@ public void pass_the_check_when_method_contains_nullable_and_not_nullable_parame
         assertTrue(passed);
     }
 
-    @Test(expected = NullPointerException.class)
+    @Test(expected = IllegalStateException.class)
```

Comment:
Please describe this behaviuor somewhere in the JavaDocs of the `NullToleranceTest` class.